Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tempfile"
-version = "3.0.9"
+version = "3.0.10"
```

Comment:
Or `3.1.0` ?

Reply:
Since we're bumping our minimum supported version I think `3.1.0` would be preferable. What do you think @Stebalien?

Reply:
Hm. Yeah, you're right. I already bumped the minimum in 3.0.9 but we should probably make minimum version bumps minor versions.